client/src/components/Subscribers/index.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { Waypoint } from 'react-waypoint';
+import { Container, Button, Spinner } from 'reactstrap';
+import noAvatar from '../../assets/noAvatar.png';
+import './style.scss';
+import Menu from '../Menu';
+import { Link } from 'react-router-dom';
+import { IUserData } from '../Profile';
+
+export interface ISubscribersProps {
+    path: any;
+    loggedId: string;
+    urlUsername: string;
+    user: IUserData;
+    page: number;
+    allSubscribersLoaded: boolean;
+    subscribers: [];
+    loaded: boolean;
+    loading: boolean;
+    loadFollow: boolean;
+    getUser: (username: string) => void;
+    getSubscribers: (loggedId: string, subscribers: string, urlUsername: string, page: number) => void;
+    changeUserFollowing: (_id: string) => void;
+    resetSubscribers: () => void;
+    followUser: (body: { _id: string }) => void;
+    unfollowUser: (body: { _id: string }) => void;
+}
+
+export class Subscribers extends React.Component<ISubscribersProps> {
+    public componentDidUpdate(prevProps: any): void {
+        if (prevProps.loggedId !== this.props.loggedId) {
+            const subscribers = this.props.path === '/profile/:username/following' ? 'following' : 'followers';
```

Comment:
Looks like hardcode. Could you check in another way?
Or use constants, or indexOf/find

Reply:
fixed